tests/unit/Makefile.am

```diff
@@ -11,6 +11,7 @@ AM_CFLAGS += -Wall -Wextra -g
 LDADD = ../../src/lib/libgpiod.la -lkmod -ludev
 
 check_PROGRAMS = gpiod-unit
+TESTS = $(check_PROGRAMS)
```

Comment:
This won't fly - the test suite works together with the gpio-mockup kernel module. By default /dev/gpiochipX requires superuser privileges. If the user doesn't have a udev rule for that, the tests will fail every time.